src/pages/filter/ui/range-panel.tsx

```diff
@@ -15,17 +16,35 @@ export interface IRangePanelFormValues {
 }
 
 const getCachedValues = () => {
-  return filterStore.readFilterCondition<IRangePanelFormValues>(
-    filterStore.selectedGroupItem.name,
-  )
+  return filterStore.readFilterCondition(filterStore.selectedGroupItem.name)
+}
+
+const getFilterValue = (arg: string) => {
+  if (isArray(getCachedValues())) {
+    const filterExpression: any[] = getCachedValues()
+
+    if (arg === 'min' && typeof filterExpression[0] === 'number') {
+      return filterExpression[0]
+    }
+
+    if (arg === 'max' && typeof filterExpression[2] === 'number') {
+      return filterExpression[2]
+    }
+
+    return ''
+  } else {
+    return arg === 'min'
+      ? getCachedValues()?.min || ''
+      : getCachedValues()?.max || ''
+  }
```

Comment:
The whole idea of filterCondition is memorizing filters between switching one to another. 
On the 129th line you can see useEffect which sets the condition when min and max change.
Probably the best way to preserve previous behavior is to normalize filters from the preset into IRangePanelFormValues interface first.